src/audio/kpb.c

```diff
@@ -1716,6 +1779,36 @@ static inline bool kpb_is_sample_width_supported(uint32_t sampling_width)
 	return ret;
 }
 
+void kpb_copy_24b_in_32b(const struct audio_stream __sparse_cache *source, uint32_t ioffset,
```

Comment:
Consider declaring this function as static if it is not going to be used outside the module.